src/kOS.Safe/Screen/IInterpreter.cs

```diff
@@ -9,6 +9,7 @@ public interface IInterpreter : IScreenBuffer
         string GetCommandHistoryAbsolute(int absoluteIndex);
         void SetInputLock(bool isLocked);
         bool IsAtStartOfCommand();
+        bool isWaitingForCommand();
```

Comment:
Arrrrg, the "is" should be capitalized here.  I just came off of doing some Java programming and had a different set of coding standards in my head.  (Typically in most Java coding standards, you capitalize types and lowercase members.)